api/app.js

```diff
@@ -22,8 +22,8 @@ const DB_USERNAME = process.env.MONGODB_USERNAME || '';
 const DB_PASSWORD = process.env.MONGODB_PASSWORD || '';
 
 // Increase post body sizing
-app.use(bodyParser.json({ limit: '25mb', extended: true }));
-app.use(bodyParser.urlencoded({ limit: '25mb', extended: true }));
+app.use(bodyParser.json({ limit: '400mb', extended: true }));
+app.use(bodyParser.urlencoded({ limit: '400mb', extended: true }));
```

Comment:
These entries are not quite the right spot to adjust file upload limits as they only restrict the size of the json or urlencoded based payloads.  FormData file uploads are handled separately from the bodyParser module.  This is why you're not seeing an effect when adjusting these values.    There are technically two spots to limit, 1) in the front end, which your earlier change covers, and 2) in the API where we actually receive the data.  We could use something like npm package `multer` in the api.  Mind you the `multer-s3` package might a be better option given what we are doing but I don't think it supports limiting filesizes.